src/radical/pilot/utils/prof_utils.py

```diff
@@ -53,8 +55,6 @@
     #
     # resources on agent nodes are consumed for all of the pilot's lifetime
     'agent' : {
```

Comment:
Did we loose `bootstrap_0_start` and `bootstrap_0_stop` in favor of `sub_agent_start` and `sub_agent_stop`? What happens if we have multiple subagents?

Reply:
> Did we loose `bootstrap_0_start` and `bootstrap_0_stop` in favor of `sub_agent_start` and `sub_agent_stop`?
No, those are still used for the full set of resources owned by the pilot.  The `sub_agent_*` are only for the resources owned by the sub agents.

> What happens if we have multiple subagents?

That works transparently :-)  Multiple event pairs lead to multiple resource accounts.